frontend/src/components/App.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import axios, { AxiosResponse } from 'axios';
+import PractitionerFeedListItem from './PractitionerFeedListItem';
+import { HapiPractitionerEntryItem, HapiPractitionersResponse } from '../types/Hapi.types';
+import { getPractitionerEmail } from '../utils/helpers';
+
+function App() {
+  const [practitioners, setPractitioners] = useState<HapiPractitionerEntryItem[]>([]);
+
+  useEffect(() => {
+    axios.get<HapiPractitionersResponse>('http://hapi.fhir.org/baseDstu3/Practitioner')
+      .then((res: AxiosResponse<HapiPractitionersResponse>) => setPractitioners(res.data.entry))
+      .catch((err) => console.error(err));
+  }, [])
+
+  return (
+    <div className="text-center">
+      <header>
+      </header>
+      <main className="bg-sky-900 text-white min-h-screen flex flex-col items-center justify-center content-center">
+        <ul>
```

Comment:
At some point, as part of a larger app, I imagine the parent element here becomes its own component. Seems fine to leave it here for now - given the purpose of the app and existing complexity.

Reply:
Oh yeah! Tbh I made a mistake going with tailwind -- I just never used it before and was curious as I've defaulted to styled components in the past. If I were to redo this assessment, I'd use React Material components library from start so I can focus on more important tasks/priorities rather than how a button should look, basic styling rules, etc.